src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -17,16 +17,13 @@ internal interface ISyntaxFactsService : ISyntaxFacts, ILanguageService
 
         bool IsPossibleTupleContext(SyntaxTree syntaxTree, int position, CancellationToken cancellationToken);
 
-        SyntaxNode Parenthesize(SyntaxNode expression, bool includeElasticTrivia = true, bool addSimplifierAnnotation = true);
```

Comment:
Two redundant methods that are no longer required on `ISyntaxFactsService`, and were not syntax facts to begin with.